Dockerfile

```diff
@@ -375,6 +375,7 @@ ENV DOCKER_BUILDTAGS="${DOCKER_BUILDTAGS}"
 WORKDIR /go/src/github.com/docker/docker
 VOLUME /var/lib/docker
 VOLUME /home/unprivilegeduser/.local/share/docker
+RUN ln -s /usr/local/bin/containerd-shim-runc-v2 /usr/local/bin/containerd-shim-runc-alias-v2
```

Comment:
I'm still a bit on the fence on this one (adding a layer just for the convenience of manually trying this construct).

Curious though; IIRC, you have an interest in using `crun` instead of `runc` - perhaps a weird idea, but would it make sense to add `crun` in the dev environment? This would also (if we want to) allow us to have CI run with `crun` instead of `runc`.